### PR TITLE
Enhances Archived Chal view

### PR DESCRIPTION
**Pull request by mk4111**
See #665

adds styling (matches previous two PRs), and adds icn

### CODE REVIEW

File: templates/partials/challenge.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
   <a href="/challenges/{{id}}" class="card__header--challenge card__data--link">
     {{title}}
+    <i class="navbar__icon icon icon--open-modal"></i>
```
Comment (JMurphyWeb) [inline templates/partials/challenge.html]: Does that icon specifically represent a modal? 

I am just wondering as `icon--challenge` seems like a more descriptive icon name?

Comment (mk4111) [inline templates/partials/challenge.html]: @JMurphyWeb - yep it's the two squares that we seen when we want to open something in a new tab.

I think 'open-modal' is a tad long, but it's an icon that's also applicable to resources and insights too, so I didn't want to call it too specific. What are your thoughts?

Comment (harrygfox) [inline templates/partials/challenge.html]: It doesn't open a modal in the traditional sense - it navigated to another endpoint. If that's the only commonality in its use then it looks like it should be 
`icon--open-link` or `icon--open-page`

Comment (mk4111) [inline templates/partials/challenge.html]: @JMurphyWeb - what do you think? :chocolate_bar:

Comment (JMurphyWeb) [inline templates/partials/challenge.html]: `icon--open-link` sounds good to me

**Comment by mk4111** [comment]
Updated the icon name! Please mahjjjj if happy peas :) <3 Tank tank

File: public/css/main.css [hunk 2]
```diff
@@ -132,6 +132,11 @@ Light-teal:     #66d9e0;
 .icon--logo {
     background-image: url("../assets/logo.png");
 }
+.icon--open-link {
+  background-image: url("../assets/open-link.svg");
+  display: inline-block;
+  margin-left: 0.4rem;
```
Comment (JMurphyWeb) [inline public/css/main.css]: Don't need to change this, but I was quite liking the icon--name classes pretty much only loading in an icon and not really adding much/any positioning attributes. Adding positioning makes these icons appearing in a different place a bit more complicated if we need to in the future/desktop views.

Comment (mk4111) [inline public/css/main.css]: @JMurphyWeb - good point, I'll keep that in mind for next time!

**Pull request merged**